src/molecules/todoList/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+// Components
+import TodoItem from '../todoItem';
+
+// Styles
+import "./TodoList.css"
+
+const TodoList = ({todoList, onTodoClick}) => {
+  
+    return (
+      <>
+        {
+            todoList.map((todoInfo , index)=>{
```

Comment:
Do not use inline function, instead define a separate function may be renderTodoItem.